lib/resources/dstu2/medication_statement.yaml

```diff
@@ -0,0 +1,287 @@
+                    "code": {
+                      "coding": [
+                        {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+                         "code": "1191",
+                         "display": "Aspirin",
+                         "userSelected": false}],
+                      "text": "aspirin"}}]}
+  url: http://hl7.org/fhir/DSTU2/domainresource-definitions.html#DomainResource.contained
+  note: It should contain medication resource with medication.code
+  children:
+
+  - name: code
+    required: 'No'
+    cardinality: 0..1
+    type: CodeableConcept
+    description:  Codes that identify this medication.
+    example: |
+               {"code": {
+                 "coding": [
+                   {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+                    "code": "1191",
+                    "display": "Aspirin",
+                    "userSelected": false}],
+                 "text": "aspirin"}}
+    url: http://hl7.org/fhir/DSTU2/medication-definitions.html#Medication.code
+    note: This code is only belongs where the resource is a medication.
+    binding:
+      description: A code that defines the type of a medication.
+      terminology:
+      - display: RxNorm
+        system: http://www.nlm.nih.gov/research/umls/rxnorm
+
+- name: dosage
+  required: 'No'
+  cardinality: 0..*
+  type: BackboneElement
+  description: How the medication is/was used by the patient.
+  note: At most one dosage can be populated.
+  example: |
+            {"dosage": [
+              {"text": "325 mg, Oral, q4hr, PRN: headache",
+               "timing": {
+                 "repeat": {
+                   "boundsPeriod": {
+                     "start": "2014-08-11T13:42:00.000-05:00"}},
+                 "code": {
+                   "text": "q4hr"}},
+               "asNeededCodeableConcept": {
+                 "coding": [
+                   {"system": "http://snomed.info/sct",
+                    "code": "25064002",
+                    "display": "headache (finding)",
+                    "userSelected": false}],
+               "text": "headache"},
+               "route": {
+                 "coding": [
+                   {"system": "http://snomed.info/sct",
+                    "code": "26643006",
+                    "display": "Oral route (qualifier value)",
+                    "userSelected": false}],
+                 "text": "Oral"},
+               "quantityQuantity": {
+                 "value": 325,
+                 "units": "mg",
+                 "system": "http://unitsofmeasure.org",
+                 "code": "mg"}}]}
+  action: create
+  children:
+
+  - name: text
+    required: 'No'
+    cardinality: 0..1
+    type: string
+    description: Free text dosage information as reported about a patient's medication use.
+    example: '{"text": "2, Oral, 2x/Wk, first dose 05/05/15 10:00:00 EDT"}'
+
+  - name: timing
+    required: 'No'
+    cardinality: 0..1
+    type: Timing
```

Comment:
I think we need to add Timing to the list of types? Its not displayed as a link.